CONTRIBUTING.md

```diff
@@ -0,0 +1,92 @@
+# How to contribute
+
+This document outlines some of the conventions on development workflow, commit
+message formatting, contact points and other resources to make it easier to get
+your contribution accepted.
+
+## Getting started
+
+- Fork the repository on GitHub.
+- Read the README.md for build instructions.
+- Play with the project, submit bugs, submit patches!
+
+## Building Dumpling & Run tests
+
+Developing Dumpling requires:
+
+* [Go 1.13.5+](http://golang.org/doc/code.html)
+* An internet connection to download the dependencies
+
+Simply run `make` to build the program.
+
+```sh
+make
+```
+
+This project contains unit tests and integration tests with coverage collection.
+See [README.md](https://github.com/pingcap/dumpling#building) for how to execute tests.
+
+### Updating dependencies
+
+Dumpling manages dependencies using [Go 1.13 module](https://github.com/golang/go/wiki/Modules).
+To add or update a dependency, either
+
+* Use the `go mod edit` command to change the dependency, or
+* Edit `go.mod` and then run `make update` to update the checksum.
```

Comment:
```suggestion
* Edit `go.mod` and then run `go mod tidy` to update the checksum.
```

dumpling doesn't have `make update` command. and in lightning it is equivalent to `go mod verify && go mod tidy` anyway.